openshift/parse-deployment-template.json

```diff
@@ -427,29 +546,95 @@
       "generate": "expression",
       "from": "[a-zA-Z0-9]{16}"
     },
-    {
-      "name": "NPM_MIRROR",
-      "displayName": "Custom NPM Mirror URL",
-      "description": "The custom NPM mirror URL",
-      "value": ""
-    },
     {
       "name": "APP_ID",
       "displayName": "App's Unique Identifier",
       "description": "A unique identifier for your app",
-      "value": ""
+      "value": "changeMe"
     },
     {
       "name": "MASTER_KEY",
       "displayName": "Parse Master Key",
       "description": "A key that overrides all permissions. Keep this secret",
-      "value": ""
+      "value": "changeMe"
```

Comment:
Should this be a variable or is this OK?

Reply:
This is representing the default value of 'changeMe' when the user imports this into openshift and clicks "process template".  They'll see this as one of the fields and hopefully get the hint to change the value.  There are committed defaults in index.js, so maybe we should remove this and fix up the index.js to fail startup if the env var is not set?

Reply:
That explanation makes sense to me, thank you.